Auth0.xcodeproj/project.pbxproj

```diff
@@ -289,6 +289,9 @@
 		970BC36C25C27095007A7745 /* Challenge.swift in Sources */ = {isa = PBXBuildFile; fileRef = 970BC36A25C27095007A7745 /* Challenge.swift */; };
 		970BC36D25C27095007A7745 /* Challenge.swift in Sources */ = {isa = PBXBuildFile; fileRef = 970BC36A25C27095007A7745 /* Challenge.swift */; };
 		970BC36E25C27095007A7745 /* Challenge.swift in Sources */ = {isa = PBXBuildFile; fileRef = 970BC36A25C27095007A7745 /* Challenge.swift */; };
+		D581CF772757D773007327D1 /* RequestSpec.swift in Sources */ = {isa = PBXBuildFile; fileRef = D581CF762757D773007327D1 /* RequestSpec.swift */; };
```

Comment:
Not sure why there's 3 entries, I added and removed the spec file a couple of times so it could be that

Reply:
There are 3 entries because the file belongs to 3 targets:

<img width="260" alt="Screen Shot 2021-12-01 at 16 23 44" src="https://user-images.githubusercontent.com/5055789/144300030-35d3b137-3b3f-4621-b198-807adfc9f483.png">